setup/steps/autoexplain_recommended_settings.go

```diff
@@ -0,0 +1,519 @@
+package steps
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/guregu/null"
+	"github.com/lib/pq"
+
+	s "github.com/pganalyze/collector/setup/state"
+	"github.com/pganalyze/collector/setup/util"
+)
+
+// N.B.: this needs to happen *after* the Postgres restart so that ALTER SYSTEM
+// recognizes these as valid configuration settings
+var ConfigureAutoExplain = &s.Step{
+	Kind:        s.AutomatedExplainStep,
+	Description: "Review auto_explain settings",
+	Check: func(state *s.SetupState) (bool, error) {
```

Comment:
It could be good to check the config settings here in `Check` as well, so we don't enter `Run` if all settings are set to recommended values.